taipy/gui/_renderers/utils.py

```diff
@@ -37,7 +37,7 @@ def _get_columns_dict_from_list(
             idx += 1
         elif col:
             _warn(
-                f'Error column "{col}" is not present in the Dataframe "{value.head(0) if hasattr(value, "head") else value}".'  # noqa: E501
+                f'Error column "{col}" is not present. Available columns: {list(value.columns)}.'  # noqa: E501
```

Comment:
that would need a check on value

Reply:
I didn't get you

Reply:
Python is very permissive and if we use the value as a dataframe in the message we'd better test that value *is* a dataframe.

Reply:
Got it